### PR TITLE
Use blueprints + cleanup build

### PR DESCRIPTION
**Pull request by joostdevries**

### CODE REVIEW

File: blueprints/twiddle.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "version": "0.3.5",
+  "dependencies": {
+    "jquery": "https://cdnjs.cloudflare.com/ajax/libs/jquery/1.9.1/jquery.js",
```
Comment (Gaurav0) [inline blueprints/twiddle.json]: 1.11.3

File: app/services/ember-cli.js [hunk 2]
```diff
@@ -1,7 +1,22 @@
...
-import config from "../config/environment";
+
+const twiddleAppName = 'demo-app';
+
+// These files will be included if not present
+const boilerPlateJs = [
+  'app',
+  'config/environment',
+  'router',
+  'initializers/router'
```
Comment (Gaurav0) [inline app/services/ember-cli.js]: This should not be exposed to the user if possible.

File: app/services/ember-cli.js [hunk 3]
```diff
@@ -90,16 +131,16 @@ export default Em.Service.extend({
...
-    var templateCode = Em.HTMLBars.precompile(code || '');
-    return this.compileJs('export default Ember.HTMLBars.template(' + templateCode + ');', moduleName);
+  compileHbs (code, filePath) {
+    let templateCode = Em.HTMLBars.precompile(code || '');
+    return this.compileJs('export default Ember.HTMLBars.template(' + templateCode + ');', filePath);
   },
 
   compileCss(code, moduleName) {
-    var prefix = "demo-app/styles/";
+    var prefix = "app/styles/";
```
Comment (Gaurav0) [inline app/services/ember-cli.js]: Why shouldn't this have the same moduleName as all other files?

File: app/services/ember-cli.js [hunk 4]
```diff
@@ -20,58 +43,75 @@ export default Em.Service.extend({
...
-      // Add config
-      out.push(this.compileJs('export default {modulePrefix:"demo-app"}', 'demo-app/config/environment'));
+      this.checkRequiredFiles(out, gist);
+      this.addBoilerPlateFiles(out, gist);
 
       // Add boot code
-      contentForAppBoot(out, {modulePrefix:'demo-app'});
+      contentForAppBoot(out, {modulePrefix: twiddleAppName});
+
+      console.log(out.join('\n'));
```
Comment (Gaurav0) [inline app/services/ember-cli.js]: Remove log

File: app/services/ember-cli.js (hunk 3 above)
Comment (joostdevries) [inline app/services/ember-cli.js]: if I read correctly the module name is actually obsolete right?

File: app/services/ember-cli.js (hunk 2 above)
Comment (joostdevries) [inline app/services/ember-cli.js]: it's not, I just want the code to live in it's own file instead of being embedded inline.

**Pull request merged**